fix create dedciated elb with eip options issue (#209)
kubernetes-sigs/cloud-provider-huaweicloud: docs/usage-guide.md

  * `bandwidth_size` Required. Specifies the bandwidth size.

>   * `bandwidth_name` Required when share_type is PER. Specifies the bandwidth name and is mandatory for Dedicated Loadbalancer if share_type is per.

Reviewer: ```suggestion
  * `bandwidth_name` Optional, it is required when `share_type=PER`. Specifies the bandwidth name.
```

Author: done in two commits as i read your message later
